CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+## Enhancements
+
+* Adds BETA support for speculative runs with `Stacks` resources, which is EXPERIMENTAL, SUBJECT TO CHANGE, and may not be available to all users by @hwatkins05-hashicorp [#1119](https://github.com/hashicorp/go-tfe/pull/1119)
+
```

Comment:
We'll also want to mention the removal of the validation for VCS presence in Stack creations here